### PR TITLE
[web] Code cleanup

### PR DESCRIPTION
**Pull request by ferhatb**
Related issue:
flutter/flutter#86891

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide] and the [C++, Objective-C, Java style guides].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt. See [testing the engine] for instructions on
writing and running engine tests.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I signed the [CLA].
- [x] All existing and new tests are passing.

### CODE REVIEW

File: lib/web_ui/test/engine/window_test.dart [hunk 1]
```diff
@@ -306,7 +306,7 @@ void testMain() {
     final Completer<bool> completer = Completer<bool>();
     final ByteData inputData = JSONMethodCodec().encodeMethodCall(MethodCall(
         'SystemChrome.setPreferredOrientations',
-        <dynamic>[]))!;
+        <dynamic>[]))!; // ignore: prefer_const_literals_to_create_immutables
```
Comment (hterkelsen) [inline lib/web_ui/test/engine/window_test.dart:309]: Why not use `const <dynamic>[]` here?

File: lib/web_ui/test/window_test.dart [hunk 2]
```diff
@@ -70,6 +70,7 @@ void testMain() {
       'flutter/navigation',
       JSONMethodCodec().encodeMethodCall(MethodCall(
         'routeUpdated',
+        // ignore: prefer_const_literals_to_create_immutables
```
Comment (hterkelsen) [inline lib/web_ui/test/window_test.dart:73]: same question here. Why not just use const literals here?

**Review by hterkelsen — APPROVED** [review]
LGTM

File: lib/web_ui/test/engine/window_test.dart (hunk 1 above)
Comment (ferhatb) [reply lib/web_ui/test/engine/window_test.dart:309]: Fixed.

File: lib/web_ui/test/window_test.dart (hunk 2 above)
Comment (ferhatb) [reply lib/web_ui/test/window_test.dart:73]: Fixed.

**Pull request merged**